scripts/azure-pipelines/osx/configuration/VagrantFile

```
@@ -0,0 +1,147 @@
+require 'json'
+
+require "erb"
+include ERB::Util
+
+configuration = JSON.parse(File.read('./vagrant-configuration.json'))
+
+servers = configuration['machine_identifiers'].map do |id|
+  {
+    :hostname => "#{configuration['base_name']}-#{id}",
+    :box => 'ramsey/macos-catalina',
+    :ram => 12000,
+    :cpu => 5
+  }
+end
+
+brew_formulas = [
+  'autoconf',
+  'automake',
+  'coreutils',
+  'gdbm',
+  'gmp',
+  'go',
+  'isl',
+  'libmpc',
+  'meson',
+  'mpfr',
+  'ninja',
+  'openssl',
+  'python',
+  'readline',
+  'sqlite',
+  'xz',
+  'yaml-cpp',
+  'yasm' ]
+
+brew_cask_formulas = [ 'powershell' ]
+
+azure_agent_url = 'https://vstsagentpackage.azureedge.net/agent/2.164.6/vsts-agent-osx-x64-2.164.6.tar.gz'
```

```suggestion
azure_agent_url = 'https://vstsagentpackage.azureedge.net/agent/2.171.1/vsts-agent-osx-x64-2.171.1.tar.gz'
```

*THAT* explains why all our agents have been getting rolled back to an old copy